extremeflash/ws.py

```diff
@@ -128,31 +154,58 @@ def boot_via_tftp(
     write_to_serial(ser, b"setenv netmask " + new_ap_netmask_str + b"\n")
     write_to_serial(ser, b"setenv serverip " + tftp_ip_str + b"\n")
     write_to_serial(ser, b"setenv gatewayip " + tftp_ip_str + b"\n")
-    write_to_serial(ser, b"tftpboot 0x2000000 " + tftp_ip_str + b":" + tftp_file.encode("ascii") + b"\n")
+    logging.info("Did setup TFTP Boot.")
+    if model == "AP3710i":
+        write_to_serial(ser, b"tftpboot 0x1000000 " + tftp_ip_str + b":" + tftp_file.encode("ascii") + b"\n")
+    elif model == "AP3935i":
+        write_to_serial(ser, b"tftpboot 0x42000000 " + tftp_ip_str + b":" + tftp_file.encode("ascii") + b"\n")
+    else:
```

Comment:
I would prefer an `elif` here and then raise an error further down to transparently fail if an unexpected model was sent to the function.